### PR TITLE
Revert PR20451 - XTimeUtils: Prevent array index out of bounds access in SystemTimeToFileTime

### PR DESCRIPTION
**Pull request by ksooo**
As discussed in #20451 this causes a serious regression.

### CODE REVIEW

File: xbmc/utils/CMakeLists.txt [hunk 1]
```diff
@@ -164,8 +164,7 @@ set(HEADERS ActorProtocol.h
             VC1BitstreamParser.h
             Vector.h
             XBMCTinyXML.h
-            XMLUtils.h
-            XTimeUtils.h)
```
Comment (ksooo) [inline xbmc/utils/CMakeLists.txt:168]: What??? Why did the PR remove XTimeUtils.h?

**Review by howie-f — APPROVED** [review]
lgtm

**Comment by howie-f** [comment]
maybe don‘t do the format changes here or apply them on the matrix pr, too.

**Comment by ksooo** [comment]
> maybe don‘t do the format changes here or apply them on the matrix pr, too.

will apply clang-format chages also on the Matrix PR

**Comment by ksooo** [comment]
jenkins build this please

**Comment by ksooo** [comment]
> I've made some formatting changes 

Can be ignored; I added a separate commit for the requested clang-format changes.

**Review by DaveTBlake — APPROVED** [review]
Thanks for this @ksooo, your button

**Pull request merged**